$OpenBSD: patch-setup_py,v 1.2 2017/11/05 16:30:52 jasper Exp $

The dependency is satisfied, stop complaining about not finding ruamel.yaml

Index: setup.py
--- setup.py.orig
+++ setup.py
@@ -33,7 +33,7 @@ setup(
     ],
     install_requires=(
         'pykwalify',
-        'ruamel.yaml<=0.15',
+        'ruamel.yaml',
         'setuptools',
     ),
     tests_require=(
